### DISCUSSION TITLE
Using LaserGRBL for inlay

### DISCUSSION THREAD

**Discussion by bucc5062** [arkypita/LaserGRBL#2231] [General]
I have been playing with and using LGRBL with my Two Tree engraver/cutter.  Though I've not done any cutting yet, I have noticed that when it engraves, it does taker(burn) a bit into the wood.

My question is if anyone has used this to do inlay where the laser etches out an area, say a square that would not cut through but allow space to drop in a square.  Most of my material is hardwood, but my concerns are how much damage could it do to the wood.  I may try some trial an error, but if anyone had done this before any input would be great.

**Comment by Tribo-2dot0** [comment]
Não sei se a tradução me fez entender bem, mas já fiz isso em mdf com um laser de 1W... O morcego claro foi recortado de uma lâmina de Marfim e encaixado na depressão aberta no mdf.
![P_20160813_112741_DF](https://github.com/arkypita/LaserGRBL/assets/83982525/9b322d7f-b416-44c1-b38a-3eaa7f77748b)
